docs/src/main/mdoc/typeclasses/bimonad.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: docs
+title:  "Bimonad"
+section: "typeclasses"
+source: "core/src/main/scala/cats/Bimonad.scala"
+scaladoc: "#cats.Bimonad"
+---
+# Bimonad
+
+The `Bimonad` trait directly extends `Monad` and `Comonad` without introducing new behaviour. `Bimonad` is 
```

Comment:
I wonder if this is confusing—even though it doesn't introduce new methods, as you note below it does have additional laws which do affect (restrict, actually) it's behavior relative to it's parent traits. Maybe prefer the word method here?

Reply:
Yep i agree. I'll use `without introducing new methods`